src/main/java/com/vestmark/bitbucket/plugin/MergeConflictDetectorServlet.java

```diff
@@ -102,9 +106,46 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
               .forEachOrdered(b -> dryRunMerge(mcd, b));
     }
     dryRunMerge(mcd, refService.getDefaultBranch(mcd.getToRepo())); // Always merge to default.
+    checkForAutoMergeFailure(mcd);
     render(response, ImmutableMap.<String, Object>of("mcd", mcd));  // Render Soy template.
   }
 
+  private void checkForAutoMergeFailure(MergeConflictDetector mcd)
+  {
+    try {
+      PullRequestSearchRequest prsReq = new PullRequestSearchRequest.Builder().state(PullRequestState.OPEN)
+          .order(PullRequestOrder.NEWEST)
+          .fromRepositoryId(mcd.getFromRepo().getId())
+          .toRepositoryId(mcd.getToRepo().getId())
+          .build();
+      long totalNPRs = pullRequestService.count(prsReq);
+      int maxNPRs = 100;
+      if (totalNPRs < maxNPRs) {
+        maxNPRs = (int) totalNPRs;
+      }
+      int begIdx = 0;
+      while (begIdx < totalNPRs) {
+        PageRequest pgReq = new PageRequestImpl(begIdx, maxNPRs).buildRestrictedPageRequest(maxNPRs);
+        Page<PullRequest> pPRs = pullRequestService.search(prsReq, pgReq);
+        boolean amfPRExists = pPRs.stream()
+            .anyMatch(
+                r -> r.getTitle().contains("Automatic merge failure") && CollectionUtils.isEmpty(r.getReviewers()));
```

Comment:
Please move this to a constant, and do we need to check if there are no reviewers?
I would think Automatic merge failure would be good enough.

Reply:
I wanted to be as strict as possible, but we can always set a rule to add someone by default to review these PRs. :) so, yes Automatic merge failure should be good enough.

Reply:
That was my thought - in case there is ever default reviewers, or even someone adds them - this PR may be unaware of that.